### ISSUE TITLE
:sparkles: Source dixa : Migrate Python CDK to Low-code CDK

### ISSUE THREAD

**Issue by RealChrisSean** [airbytehq/airbyte#29118]
https://docs.airbyte.com/integrations/sources/dixa

**Rules for the Connectors Hackathon**
You can only work on one connector at a time
Make sure to resolve all suggestions from the Airbyte team
Must include all streams of the connector you’re migrating. Otherwise, it will not be accepted.
If you don’t commit anything within five days, the issue may be assigned to someone else.

**Comment by kaggrwal** [comment]
Hi @RealChrisSean I would like to work on this, can you assign me this ?

**Comment by RealChrisSean** [comment]
Good luck!

**Comment by topefolorunso** [comment]
@RealChrisSean Can you assign this to me since no activity for 7 days?

**Comment by kaggrwal** [comment]
Hey @topefolorunso , i'm working on it. I was busy with some other work.

**Comment by topefolorunso** [comment]
Alright, buddy! Good luck with it!

**Comment by kaggrwal** [comment]
Hi @RealChrisSean , can i get any test account or any sample token to test the DIXA apis. 

as per their documentation, it is not possible to generate a new token now

`In order to get access to the Dixa API, you have to create an API token as a form of authentication in Dixa. Currently, only administrators in Dixa terms have access to generating such a token. If you are an agent, you will not have the permission to authenticate and therefore use the Dixa API.

If you are an admin, you can easily create a token in our Integrations page. Here is a tutorial on how to do so:`

**Comment by RealChrisSean** [comment]
Hello we will work on getting these credentials. It may take a few days.

**Comment by kaggrwal** [comment]
@RealChrisSean  no problem, i will wait :)

**Comment by RealChrisSean** [comment]
Please feel free to work on another issue while we wait on this. We have a back log at the moment so may take a few more days :(

**Comment by justinbchau** [comment]
you got it! @topefolorunso
